Demo/Demo/Networking/DemoMerchantAPI.swift

```diff
@@ -185,4 +216,42 @@ final class DemoMerchantAPI {
         }
         return request
     }
+    
+    private func createSetupTokenUrlRequest(
+        setupTokenRequest: SetUpTokenRequest, environment: Demo.Environment, selectedMerchantIntegration: MerchantIntegration
+    ) throws -> URLRequest {
+        var completeUrl = environment.baseURL
+       
+        completeUrl += selectedMerchantIntegration.path
+        completeUrl.append(contentsOf: setupTokenRequest.path)
+        guard let url = URL(string: completeUrl) else {
+            throw URLResponseError.invalidURL
+        }
+        var request = URLRequest(url: url)
+        request.httpMethod = setupTokenRequest.method.rawValue
+        request.httpBody = setupTokenRequest.body
+        setupTokenRequest.headers.forEach { key, value in
+            request.addValue(value, forHTTPHeaderField: key.rawValue)
+        }
+        return request
+    }
+    
+    private func createPaymentTokenUrlRequest(
+        paymentTokenRequest: PaymentTokenRequest, environment: Demo.Environment, selectedMerchantIntegration: MerchantIntegration
```

Comment:
The reason for all this redundancy is because we will be deleting APIRequest protocol for another. I will add conformance so all the REST API calls can use one function.